tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -15406,3 +15406,159 @@ TEST_F(ZoneUnitarySysTest, UnitarySystemModel_MultiSpeedDXCoilsDirectSolutionTes
+         "  Cooling Coil Setpoint Manager,  !- Name",
+         "  Temperature,                    !- Control Variable",
+         "  Always 20C,                     !- Schedule Name",
+         "  Zone 2 Inlet Node;              !- Setpoint Node or NodeList Name",
+
+         "Curve:Quadratic,",
+         "  Quadratic,                      !- Name",
+         "  0.8,                            !- Coefficient1 Constant",
+         "  0.2,                            !- Coefficient2 x",
+         "  0.0,                            !- Coefficient3 x**2",
+         "  0.5,                            !- Minimum Value of x",
+         "  1.5;                            !- Maximum Value of x",
+
+         "Curve:Biquadratic,",
+         "  Biquadratic,                    !- Name",
+         "  0.942587793,                    !- Coefficient1 Constant",
+         "  0.009543347,                    !- Coefficient2 x",
+         "  0.000683770,                    !- Coefficient3 x**2",
+         "  -0.011042676,                   !- Coefficient4 y",
+         "  0.000005249,                    !- Coefficient5 y**2",
+         "  -0.000009720,                   !- Coefficient6 x*y",
+         "  12.77778,                       !- Minimum Value of x",
+         "  23.88889,                       !- Maximum Value of x",
+         "  18.0,                           !- Minimum Value of y",
+         "  46.11111,                       !- Maximum Value of y",
+         "  ,                               !- Minimum Curve Output",
+         "  ,                               !- Maximum Curve Output",
+         "  Temperature,                    !- Input Unit Type for X",
+         "  Temperature,                    !- Input Unit Type for Y",
+         "  Dimensionless;                  !- Output Unit Type",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects)); // read idf objects
+
+    // call the UnitarySystem factory
+    std::string compName = "UNITARY SYSTEM MODEL";
+    bool zoneEquipment = true;
+    UnitarySystems::UnitarySys::factory(state, DataHVACGlobals::UnitarySys_AnyCoilType, compName, zoneEquipment, 0);
+    UnitarySystems::UnitarySys *thisSys = &state.dataUnitarySystems->unitarySys[0];
+
+    DataGlobals::NumOfTimeStepInHour = 1;
+    DataGlobals::MinutesPerTimeStep = 60;
+    DataZoneEquipment::ZoneEquipInputsFilled = true;                                    // indicate zone data is available
+    thisSys->getUnitarySystemInputData(state, compName, zoneEquipment, 0, ErrorsFound); // get UnitarySystem input from object above
+    std::string const error_string = delimited_string({
+        "   ** Severe  ** AirLoopHVAC:UnitarySystem = UNITARY SYSTEM MODEL\n   **   ~~~   ** For FAN:ONOFF = SUPPLY FAN 1\n   **   ~~~   ** Fan operating mode must be continuous (fan operating mode schedule values > 0).\n   **   ~~~   ** Error found in Supply Air Fan Operating Mode Schedule Name ALWAYSZERO\n   **   ~~~   ** ...schedule values must be (>0., <=1.)"
```

Comment:
@matthew-larson are you clang formatting as you go? It seems this line would have wrapped. Not sure if it's worth another push if everything else is OK.

Reply:
I have clang formatting ON in CLion.  I can double check that it's working right.